docker-compose.yml

```diff
@@ -8,19 +8,24 @@ services:
         - http_proxy
         - https_proxy
         - no_proxy
-    image: hmctspublic.azurecr.io/spring-boot/template
+    image: hmcts/ccd-message-publisher
+    container_name: ccd-message-publisher
     environment:
-      # these environment variables are used by java-logging library
-      - ROOT_APPENDER
-      - JSON_CONSOLE_PRETTY_PRINT
-      - ROOT_LOGGING_LEVEL
-      - REFORM_SERVICE_TYPE
-      - REFORM_SERVICE_NAME
-      - REFORM_TEAM
-      - REFORM_ENVIRONMENT
-      - LOGBACK_DATE_FORMAT
-      - LOGBACK_REQUIRE_THREAD
-      - LOGBACK_REQUIRE_ALERT_LEVEL=false
-      - LOGBACK_REQUIRE_ERROR_CODE=false
+      DATA_STORE_DB_HOST: ccd-shared-database
+      DATA_STORE_DB_PORT: 5432
+      DATA_STORE_DB_USERNAME: "${DB_USERNAME}"
+      DATA_STORE_DB_PASSWORD: "${DB_PASSWORD}"
+      CCD_CASE_EVENTS_DESTINATION: ccd-case-events
+      SERVICE_BUS_CONNECTION_STRING: <AzureServiceBusConnectionString>
+      # Comment below line to use real Azure Service Bus
+      SPRING_PROFILES_ACTIVE: dev
+      # For debug
+      # JAVA_TOOL_OPTIONS: "-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=*:5005"
     ports:
       - $SERVER_PORT:$SERVER_PORT
+    networks:
+      - ccd-network
```

Comment:
Ah I just tried that but can't add it in since the service isn't known in the context of just this repo. However have added it to accompanying ccd-docker changes - https://github.com/hmcts/ccd-docker/pull/114